dluhc-component-library/src/main.tsx

```diff
@@ -18,7 +19,23 @@ const renderSiteSelectionForm: renderFunction = (
   render(<SiteSelectionForm />, element);
 };
 
+const renderMap: renderFunction = (_options: {}, element: HTMLElement) => {
+  render(
+    <MapContainer>
+      <BaseMap
+        mapId="site-selection-map"
+        lat={54.97}
```

Comment:
`Non-Blocker` Doesnt need to be part of this PR but is it worth pulling this out into the render function options?

Reply:
Will do this when we have a more specific map implementation. This render function will probably be removed in the near future.